.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
good job adding the node modules to the gitignore!

Reply:
Thanks!